examples/simple_xo/src/main.rs

```diff
@@ -41,7 +41,7 @@ fn main() {
     let merkle_state = MerkleState::new(db.clone());
     let orig_root = merkle_db.get_merkle_root();
 
-    let signer = HashSigner::new(vec![00u8, 1, 2]);
+    let signer = HashSigner;
```

Comment:
Now that we can avoid it, we should take HashSigner out of this example - we only had it here out of necessity.